scripts/rabbitmq-server

```diff
@@ -160,9 +160,7 @@ if [ "${RABBITMQ_CONFIG_FILE_NOEX}.conf" = "${RABBITMQ_CONFIG_FILE}" ]; then
         mkdir -p "${RABBITMQ_GENERATED_CONFIG_DIR}"
     fi
 
-    if [ ! -f "${RABBITMQ_SCHEMA_DIR}/rabbit.schema" ]; then
-        cp "${RABBITMQ_HOME}/priv/schema/rabbit.schema" "${RABBITMQ_SCHEMA_DIR}"
-    fi
+    cp -f "${RABBITMQ_HOME}/priv/schema/rabbit.schema" "${RABBITMQ_SCHEMA_DIR}"
```

Comment:
I confirmed that `-f` is named consistently and seems to advertise the same behavior for Linux, MacOS and FreeBSD.

Reply:
yep! at least one command is consistent